client/src/pages/private/ParticipantTable.js

```diff
@@ -272,17 +284,25 @@ const ParticipantTable = () => {
       case 'archive':
         return (
           <>
-            {!row.status.includes('withdrawn') && (
-              <Button
-                onClick={(event) => {
-                  setActionMenuParticipant(row.engage);
-                  setAnchorElement(event.currentTarget);
-                }}
-                variant='outlined'
-                size='small'
-                text='Actions'
-              />
-            )}
+            {!row.status.includes('withdrawn') &&
```

Comment:
Considering `Change site` button, isn't it better to make `Archive` a sub menu of `Actions`?

Reply:
This will be fixed by HCAP-1120